node/cmd/root.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/certusone/wormhole/node/cmd/ccq"
 	"github.com/certusone/wormhole/node/cmd/debug"
 	"github.com/certusone/wormhole/node/cmd/spy"
+	transferverifier "github.com/certusone/wormhole/node/cmd/transfer-verifier"
```

Comment:
It is [go convention](https://go.dev/blog/package-names) not to use dashes or underscores, e.g. guardiansigner. Please match your folders to your package name.